pkgs/servers/nosql/ferretdb/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+, tree
```

Comment:
Probably not used :thinking:

Reply:
You are right :thinking: it is leftover from debugging.